### ISSUE TITLE
Protocol description

### ISSUE THREAD

**Issue by abitrolly** [mattn/ft#4]
How does `ft` transfer files? It is possible to create Android version?

**Comment by mattn** [comment]
Yes, it's possible. but I don't have interesting now about porting to Android.

**Comment by abitrolly** [comment]
So what about protocol description so that other people can try?
